api/books.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "axios"
+
+type GetBooksProps = {
```

Comment:
For `type` vs `interface` -> I would suggest using `type` for objects that are outside your control. E.g. API types

`interface` for props

Reply:
Thanks @normancapule !